src/styles/blocks/label.scss

```diff
@@ -0,0 +1,22 @@
+.label {
+  display: flex;
+  fill: orangered;
+  align-items: center;
+
+  &__svg {
+    display: flex;
+    justify-content: start;
+    align-items: flex-start;
+    width: 50px;
+    height: 50px;
```

Comment:
Looks like square, you can create mixin for it, I saw that you've used this logic in wrapper (and in circle mixin)